### PR TITLE
Do not merge until 8/9: Class Settings in Graph

### PR DESCRIPTION
**Pull request by vardhan1990**

### CODE REVIEW

File: api-reference/beta/resources/teamClassSettings.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# teamClassSettings resource type
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+Settings to configure class-type [teams](team.md). 
+Note: Class settings are open data type. It is expected for this property to be absent for teams that are not of type class. Also, this property will be absent if the team admin of a class team has not interacted with the new setting.
```
Comment (AkJo) [inline api-reference/beta/resources/teamClassSettings.md]: Class settings in not an "open data type". The Team entity is being extended to be an OpenType. ClassSettings is a complex type.

File: api-reference/beta/resources/teamClassSettings.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## teamClassSettings Properties
+| Property	   | Type	|Description|
+|:---------------|:--------|:----------|
+|notifyGuardiansAboutAssignments|Boolean|If set to true, teachers in classes have enabled notifications for assignemnts to guardians of students in that class.|
```
Comment (AkJo) [inline api-reference/beta/resources/teamClassSettings.md]: typo: "assignemnts "

**Comment by Lauragra** [comment]
@vardhan1990 , what is the status of this PR? Is it something that we need to get merged in? Please note that we have a new process and requirements for PR. If you want to merge this, please close this PR and open a new one based on the master branch. Please also respond to review feedback. Thanks.

**Comment by Lauragra** [comment]
@dipakboyed , @mmast-msft , this PR was never published and cannot be merged because it was opened before the migration. Please create a new PR if we still need to publish this documentation.

**Pull request closed without merging**